bc_access_menu_allocations/manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: Allocation
+category: ACCESS
+description: "Allocation\r\n"
+icon: 'fas://magic'
```

Comment:
We should probably change these icons to something that we at least think will work - I'm sure they'll have feedback on that.

Here's my guess as to which should be what.

metrics -> fas://signal
allocations -> fas://usd
allocations -> fas://question
allocations -> fas://cog